dev/DynamicDependency/DataStore.h

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#pragma once
+
+#include "PackageDependency.h"
+
+namespace MddCore
+{
+    class DataStore
+    {
+    public:
+        DataStore() = delete;
+        ~DataStore() = delete;
+
+    public:
+        static constexpr PCWSTR fileExtension{ L".mdd" };
```

Comment:
Nit: Using the "string_view_literals" namespace lets you make this be of type wstring_view by appending "sv" to the string like `L".mdd"sv`. Helps reduces raw pointers in the codebase and the string length is computed at compile time.

Reply:
Will consider in future PR